test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_config_test.clj

```diff
@@ -444,15 +446,23 @@
         (is (thrown-with-msg?
               IllegalArgumentException
               #"Invalid script string in webserver 'post-config-script' configuration"
-              (jetty9/start-webserver!
-                (jetty9/initialize-context)
-                (merge base-config
-                       {:post-config-script (str "AHAHHHGHAHAHAHEASD!  OMG!")})))))
+              (let [context (jetty9/initialize-context)]
+                (try
+                  (jetty9/start-webserver!
+                   context
+                   (merge base-config
+                          {:post-config-script (str "AHAHHHGHAHAHAHEASD!  OMG!")}))
+                  (finally
+                    (jetty9/shutdown context)))))))
       (testing "Throws an error if the script can't be executed."
         (is (thrown-with-msg?
               IllegalArgumentException
               #"Invalid script string in webserver 'post-config-script' configuration"
-              (jetty9/start-webserver!
-                (jetty9/initialize-context)
-                (merge base-config
-                       {:post-config-script (str "Object x = null; x.toString();")}))))))))
+              (let [context (jetty9/initialize-context)]
+                (try
+                  (jetty9/start-webserver!
+                   context
+                   (merge base-config
+                          {:post-config-script (str "Object x = null; x.toString();")}))
+                  (finally
+                    (jetty9/shutdown context))))))))))
```

Comment:
There's a couple new wolfe principle violations here and from the `testing` block above this one, probably due to the tk logging update.